common/src/main/java/com/hartwig/serve/common/classification/EventClassifierConfig.java

```diff
@@ -110,6 +110,15 @@ public abstract class EventClassifierConfig {
     @NotNull
     public abstract Set<String> ebvPositiveEvents();
 
+    @NotNull
+    public abstract Set<String> positiveEvents();
+    @NotNull
+    public abstract Set<String> positiveBlacklistEvents();
+    @NotNull
+    public abstract Set<String> negativeEvents();
+    @NotNull
+    public abstract Set<String> negativeBlacklistEvents();
+
```

Comment:
Minor formatting: adding some newlines would make this consistent with rest of the file